sui-execution/v0/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -359,6 +359,7 @@ mod checked {
             command_kind: CommandKind<'_>,
```

Comment:
I think we can revert these changes to v0?

Reply:
not sure what this means?